src/org-manager/containers/overview-pending/pending-overview.component.ts

```diff
@@ -31,7 +35,12 @@ export class OverviewPendingComponent implements OnInit {
       } else {
         this.store.dispatch(new fromRoot.Go({ path: ['/'] }));
       }
+      this.errorMessages = [];
+      if (pendingOrgs$.errorMessage) {
+        this.errorMessages.push({message: pendingOrgs$.errorMessage});
```

Comment:
there is no need for this.errorMessage local array.... u can create osbervable and use async pipe direclty in html component... this will save u few lines of code... ask Adnan or Toiresa they know how to do it
also if u subscribing u need to unsubscribe on destroy but no need to do that if u use async pipe

Reply:
ok

Reply:
Did both the changes now. Please have a look @fazanki